src/app/login/page.tsx

```diff
@@ -0,0 +1,164 @@
+"use client";
+import Button from "@/components/Button";
+import Image from "next/image";
+import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import { useStore } from "../store";
+const toastr = require("toastr");
+
+const Login = () => {
+	/* Hooks ----------------------------------------------------------------------------------------------------------------------------------------------------------------*/
```

Comment:
No need to add these comments I believe. It might be too redundant. Open to discuss!

Reply:
Sounds good to me! Joel made me add them at one point so I got used to it but I agree it can be too redundant for small components like these